gigascience/gigadb-website
# Add help box for dataset table filtering

# Pull request for issue: #1711

This is a pull request for the following functionalities:

* http://gigadb.gigasciencejournal.com/adminDataset/admin Wrapped table instructions in a help box 
![image](https://github.com/gigascience/gigadb-website/assets/143437854/1b93742d-31f7-4d6f-8174-3284be1e8f80)


NOTE: the styles of this type of table are revamped in https://github.com/gigascience/gigadb-website/pull/1701
Once this PR is merged, the styling for this panel should be updated too

## How to test?

Go to http://gigadb.gigasciencejournal.com/adminDataset/admin and verify design and layout, interact with the keyboard to verify that the panel toggles

## How have functionalities been implemented?

The panel styles were reused to add a help box, for the toggle interaction, a similar widget in the search page was used as reference (protected/views/search/_filter.php)

## Any issues with implementation?

--

## Any changes to automated tests?

--
